src/link.cpp

```diff
@@ -1814,10 +1814,14 @@ static void construct_linker_job_elf(LinkJob *lj) {
         lj->args.append("--allow-shlib-undefined");
     }
 
-    if (g->zig_target->os == OsZen) {
+    // MIPS entry point name is __start instead of _start, force the linker to
```

Comment:
I'm fine with this change, but, should we instead embrace the MIPS convention of __start and solve this in start.zig? https://github.com/ziglang/zig/blob/dc3c4197c6e6564fb2558768cf2102b868347442/std/special/start.zig#L26

Reply:
It makes sense for compatibility sake, I'll change this.